sections/Dataset.tex

```diff
@@ -0,0 +1,14 @@
+\section{Dataset}
+Nordjyske is a media group maintaining many local newspapers, radios and websites in the North Jutland region of Denmark.
+The 'Nordjyske' dataset consists of news articles from 2017-2019. 
```

Comment:
Maybe we should clarify that this is a dataset we "created" / "selected" not one that has been published or anything.

Reply:
also its only a small subset of the entire 'nordjyske' dataset, but we can mention some of the reasons we chosse 2017-2019, file types, covid break, etc.